content/zh/docs/concepts/workloads/controllers/cron-jobs.md

```diff
@@ -85,6 +85,54 @@ This example CronJob manifest prints the current time and a hello message every
 [使用 CronJob 运行自动化任务](/zh/docs/tasks/job/automated-tasks-with-cron-jobs/)
 一文会为你详细讲解此例。
 
+<!--
+### Cron schedule syntax
+-->
+### Cron 时间表语法
+
+```
+# ┌───────────── 分钟 (0 - 59)
+# │ ┌───────────── 小时 (0 - 23)
+# │ │ ┌───────────── 月的某天 (1 - 31)
+# │ │ │ ┌───────────── 月份 (1 - 12)
+# │ │ │ │ ┌───────────── 周的某天 (0 - 6) （周日到周一；在某些系统上，7 也是星期日）
+# │ │ │ │ │                                   
+# │ │ │ │ │
+# │ │ │ │ │
+# * * * * *
+```
+
+
+<!-- 
+| Entry 										| Description																									| Equivalent to |
+| ------------- 						| ------------- 																							|-------------  |
+| @yearly (or @annually)		| Run once a year at midnight of 1 January										| 0 0 1 1 * 		|
+| @monthly 									| Run once a month at midnight of the first day of the month	| 0 0 1 * * 		|
+| @weekly 									| Run once a week at midnight on Sunday morning								| 0 0 * * 0 		|
+| @daily (or @midnight)			| Run once a day at midnight																	| 0 0 * * * 		|
+| @hourly 									| Run once an hour at the beginning of the hour								| 0 * * * * 		|
+-->
+| 输入                    | 描述                            | 相当于         |
+| -------------           | -------------                  |-------------   |
+| @ 每年                  | 每年 1 月 1 日的午夜运行一次      | 0 0 1 1 *      |
+| @ 每月                  | 每月第一天的午夜运行一次          | 0 0 1 * *      |
+| @ 每周                  | 每周的周日午夜运行一次            | 0 0 * * 0      |
+| @ 每天（或者 @ 午夜）    | 每天午夜运行一次                  | 0 0 * * *      |
+| @ 每小时                | 每小时的开始一次                  | 0 * * * *      |
```

Comment:
The first column is literals and should not be translated.

Reply:
copy